cmake/configs/nuttx_px4fmu-v2_default.cmake

```diff
@@ -24,14 +24,15 @@ set(config_module_list
 	#drivers/hott
 	#drivers/iridiumsbd
 	#drivers/irlock
-	drivers/imu/l3gd20
+        drivers/imu/l3gd20
 	drivers/imu/lsm303d
 	drivers/magnetometer/hmc5883
 	drivers/magnetometer/lis3mdl
 	#drivers/mb12xx
 	#drivers/mkblctrl
-	drivers/imu/mpu6000
-	drivers/imu/mpu9250
+        drivers/imu/mpu6000
```

Comment:
Can you undo these whitespace changes? PX4 Firmware generally uses tabs everywhere. See the astyle comment below.

Reply:
Yes this is exactly tab from Qt 5.5.1. But I will check again later to prevent this